src/components/HomePage/SearchArea/SearchArea.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+import { connect } from "react-redux";
+import { addSnackbar } from "../../../actions/notificationActions";
+
+import useToggle from "../../../hooks/useToggle";
+
+import JOB_TYPES from "./jobTypes";
+
+import {
+    FormControl,
+    Typography,
+    Paper,
+    Slider,
+    TextField,
+    MenuItem,
+    Collapse,
+} from "@material-ui/core";
+
+import { makeStyles } from "@material-ui/styles";
+import searchAreaStyle from "./SearchArea.module.css";
+
+import SearchBar from "./SearchBar";
+import ShowAdvancedOptionsButton from "./ShowAdvancedOptionsButton";
+import SliderValueTooltip from "./SliderValueTooltip";
+
+const INITIAL_JOB_TYPE = "";
+const INITIAL_JOB_DURATION = 1;
```

Comment:
We are mixing things, these values are for rendering purposes, regarding `INITIAL_JOB_TYPE`, it can start as `undefined`, but `INITIAL_JOB_DURATION` must have a value, as the slider needs it to render. However, that filter logic can and should be done in the `submitForm` function, that will handle the API calling and fetching the search results

Reply:
I've put a comment in the function to remind the future implementer of this functionality about the filter nature of the fields, which are optional